albertoecf/tdd_web
# feat:  Add Offer Model, Migration, and Template for Gotter App

## Description
This pull request adds new functionality to the Gotter App. It includes the following changes:

- Added settings to IDE
- Improved FT Add Offer
- Registered the app and implemented a template
- Refactored the main page with a template
- Added a table in HTML
- Added tests for the post method and DRY functional test
- Added an Offer model for offers and its migrations
- Prevented blank items from being saved in every request

These changes add an Offer model to the app with a migration, text field, and a template for the home page. The tests were also updated to reflect the new functionality.

## How to test it

- Run the tests to ensure they all pass
- Verify that the settings for the IDE have been added
- Verify that the main page has been refactored with a template and a table has been added
- Verify that the app has been registered and the template for the home page has been implemented
- Verify that the Offer model has been added with a migration and a text field
- Verify that blank items are not saved in every request.
